sdk/communication/azure-communication-identity/azure/communication/identity/__init__.py

```diff
@@ -16,6 +16,8 @@
     identifier_from_raw_id,
     MicrosoftTeamsUserIdentifier,
     MicrosoftTeamsUserProperties,
+    MicrosoftTeamsAppIdentifier,
```

Comment:
expose them for other ACS SDKs those expose all identifiers

Reply:
Exposed.